src/rtf-plugins/fixture-managers/yarpmanager/YarpFixManager.cpp

```diff
@@ -50,8 +51,14 @@ bool YarpFixManager::setup(int argc, char** argv) {
 
         fixtureName = rf.find("fixture").asString();
         std::string appfile = rf.findFileByName(std::string("fixtures/"+fixtureName).c_str());
+        if (!appfile.size())
+        {
+            yInfo("trying to load from absolute path");
```

Comment:
This print is a bit out of context without more information, for example the class from which it is emitted.